objects/69ead2/contained/DUESETUP/script.lua

```diff
@@ -408,29 +402,33 @@ function g3CardsSelected()
     return count
 end
 
-function g3Plus(card)
+function g3Plus(card, color)
     local selected = g3CardsSelected()
     local guid = card.guid
     if impendTable[guid].energy[3] == 1 then
         impendTable[guid].energy[3] = 0
     elseif impendTable[guid].energy[3] == -1 or selected < 2 then
         impendTable[guid].energy[3] = 1
+    else
+        broadcastToColor("You have already made your two selections for G3 energy adjustments!", color, {r=1, g=1, b=0})
```

Comment:
nit: `Player[color].broadcast(...)` is preferable. i can't remember why, but there was some oddity in the past i think i encountered which is why i switched the mod over to that way